frontend/src/Store/Actions/releaseActions.js

```diff
@@ -44,7 +44,7 @@ export const defaultState = {
         return 10000;
       }
 
-      return item.languages[0].id;
+      return item.languages[0]?.id ?? 0;
```

Comment:
Is this solved by the fixed to DownloadDecisionMaker, or are their other cases of no language?